app_legacy/Helpers/database/set-claim.php

```diff
@@ -15,39 +16,36 @@
  */
 function insertClaim(string $user, int $gameID, int $claimType, int $setType, int $special, int $permissions): bool
```

Comment:
Refactored `insertClaim` and `getActiveClaimCount` while at it because I thought there's an issue in those because of the minutes remaining calculation.